akka-persistence-query/src/main/scala/akka/persistence/query/journal/leveldb/LeveldbReadJournal.scala

```diff
@@ -31,12 +32,21 @@ class LeveldbReadJournal(system: ExtendedActorSystem, config: Config) extends sc
 
   override def query[T, M](q: Query[T, M], hints: Hint*): Source[T, M] = q match {
     case EventsByPersistenceId(pid, from, to) ⇒ eventsByPersistenceId(pid, from, to, hints)
+    case AllPersistenceIds                    ⇒ allPersistenceIds(hints)
     case unknown                              ⇒ unsupportedQueryType(unknown)
   }
 
   def eventsByPersistenceId(persistenceId: String, fromSeqNr: Long, toSeqNr: Long, hints: Seq[Hint]): Source[EventEnvelope, Unit] = {
     Source.actorPublisher[EventEnvelope](EventsByPersistenceIdPublisher.props(persistenceId, fromSeqNr, toSeqNr,
       refreshInterval(hints), maxBufSize, writeJournalPluginId)).mapMaterializedValue(_ ⇒ ())
+      .named("eventsByPersistenceId-" + persistenceId)
+  }
+
+  def allPersistenceIds(hints: Seq[Hint]): Source[String, Unit] = {
+    val liveQuery = refreshInterval(hints).isDefined
```

Comment:
Slightly weird (in a good way I guess, since we can support "push" from the journal), should be documented that the actual polling interval is ignored.